charts/generic-service/README.md

```diff
@@ -152,7 +152,8 @@ generic-service:
   replicaCount: 2
 
   ingress:
-    host: project-name-dev.hmpps.service.justice.gov.uk
+    hosts:
+      - project-name-dev.hmpps.service.justice.gov.uk
```

Comment:
Could you keep the 'host' property for backwards compatibility?

Reply:
What I meant was that you could have both the original 'host' and new 'hosts' properties (with more complex processing logic). Though if you are thinking of making changes in the dependent charts this is not an issue 🙂